rotkehlchen/accounting/structures/base.py

```diff
@@ -178,13 +181,13 @@ def serialized_event_identifier(self) -> str:
         return '0x' + hex_representation
 
     @classmethod
-    def deserialize_event_identifier_from_kraken(cls, val: str) -> bytes:
-        """Takes a `refid` from Kraken exchange and turns it into a bytes event_identifier."""
+    def deserialize_event_identifier_from_str(cls, val: str) -> bytes:
+        """Takes an arbitrary string identifier and turns it into a bytes event_identifier."""
         return val.encode()
 
     @classmethod
-    def deserialize_event_identifier(cls, val: str) -> bytes:
-        """Takes an abritrary string and turns it into a bytes event_identifier.
+    def deserialize_event_identifier_from_tx_hash(cls, val: str) -> bytes:
+        """Takes a tx_hash and turns it into a bytes event_identifier.
         May raise:
         - DeserializationError if value is not valid transaction hash.
         """
```

Comment:
This name is confusing at least for me since we have `EVMTxHash`. But this is valid for any string, Why not restore to its original name?